extensions/sql-database-projects/package.vscode.json

```diff
@@ -0,0 +1,7 @@
+{
+	"name": "sql-database-projects-vscode",
+	"extensionDependencies": [
+		"ms-mssql.mssql",
```

Comment:
why doesn't the mssql extension name also start with Microsoft?

Reply:
No idea, that's just the publisher ID we have. https://github.com/microsoft/vscode-mssql/blob/main/package.json#L6